src/skmultiflow/trees/nodes/active_learning_node_for_regression.py

```diff
@@ -30,7 +30,7 @@ def learn_from_instance(self, X, y, weight, ht):
             Instance target value.
         weight: float
             Instance weight.
-        ht: HoeffdingTree
+        ht: HoeffdingTreeClassifier
```

Comment:
Here, I believe that `HoeffdingTreeRegressor` would be the correct option.

Reply:
Updated in 38da85c44c8c524c58704ec9e8718c554e313b0f